bogota-abril-2019/css-magazine-cover/JuanChaparro/styles.css

```diff
@@ -0,0 +1,68 @@
+#container {
+    border: 50px solid #ED1C24;
+    width: 1000px;  
+    margin: auto;
+}
+
+#main-container {
```

Comment:
Avoid use of id selector. Usually is a code smell

Reply:
I corrected it, i have a question.
When should I use id selector?